### PR TITLE
adding docker support

### PR DESCRIPTION
**Pull request by nhatfield**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,3 +28,47 @@ node js/bin/metricly.js
...
+// Build and run / develop as a docker container
+docker build -t metricly-cli .
+
+// Run it as a docker command
+docker run -it -v ~/.metricly-cli.json:/root/.metricly-cli.json:rw metricly-cli
+
+// Use it as an alias. Edit ~/.bash_profile and add:
+alias metricly-cli="docker run -it -v ~/.metricly-cli.json:/root/.metricly-cli.json:rw metricly-cli"
+
+$ metricly-cli
```
Comment (TheConnMan) [inline README.md]: This is going to get out of date real quick. I recommend not adding it.

**Comment by nhatfield** [comment]
Correct, avoid installing locally, add support for Docker while keeping the Dockerfile in source, ultimately having Travis-CI build the docker container and push to dockerhub. With an official docker image it can be used on coreos

File: entrypoint.sh [hunk 2]
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```
Comment (TheConnMan) [inline entrypoint.sh]: What's the benefit of having this file and not calling this command directly?

**Comment by TheConnMan** [comment]
@nhatfield found that the binaries suit his need and said this could be closed.

**Pull request closed without merging**